packages/unlock-js/src/PublicLock/v14/index.js

```diff
@@ -0,0 +1,8 @@
+import abis from '../../abis'
+import v13 from '../v13'
+
+export default {
+  ...v13,
+  version: 'v14',
+  PublicLock: abis.PublicLock.v14,
```

Comment:
No function change?

Reply:
we added a `setExpirationKey` but I wasn't sure we'll need it in unlockjs rn